style.css

```diff
@@ -82,6 +109,9 @@ form.example button {
   width: 310px;
   height: 610px;
   right: 0;
+  top: 0;
+  border: 1px solid black;
+  margin-top: 60px;
```

Comment:
You've already used a margin-top and border property, please use once.

Reply:
fix